Bottom: 3319b672447556ce3dcc15cb07a6937d76e88aab
Top:    2dbd0fafd50d0b0ed83c40787d65426b46a4254a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-28 22:49:22 +0100

Refresh of determine-layer-count-in

---

diff --git a/src/misc.c b/src/misc.c
index bc27338..40cb58b 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -995,8 +995,6 @@ ParseGroupString (char *group_string, LayerGroupType *LayerGroup, int *LayerN)
     s_set = false;              /* provide a default setting for old formats */
   int groupnum[MAX_LAYER + 2];
 
-<<<<<<< current
-=======
   *LayerN = 0;
 
   /* Deterimine the maximum layer number */
@@ -1037,7 +1035,6 @@ ParseGroupString (char *group_string, LayerGroupType *LayerGroup, int *LayerN)
 
   printf ("ParseGroupString (\"%s\", %p, %p) - found %i layers\n", group_string, LayerGroup, LayerN, *LayerN);
 
->>>>>>> patched
   /* clear struct */
   memset (LayerGroup, 0, sizeof (LayerGroupType));
 
diff --git a/src/parse_y.y b/src/parse_y.y
index 0992b0c..dcafe11 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -213,12 +213,14 @@ parsepcb
 		| {
 		    if (yyPCB != NULL)
 		      {
-			/* This is the case where we load a footprint with file->open, or from the command line */
+printf ("FOO\n");
+			/* This case is when we load a footprint with file->open, or from the command line */
 			yyFont = &yyPCB->Font;
 			yyData = yyPCB->Data;
 			yyData->pcb = yyPCB;
 			yyData->LayerN = 0;
-		      }
+		      //}
+		      } else printf ("YO\n");
 		  }
 		  element
 		  {
@@ -226,7 +228,8 @@ parsepcb
 		    ElementType *e;
 		    if (yyPCB != NULL)
 		      {
-			/* This is the case where we load a footprint with file->open, or from the command line */
+printf ("BAR\n");
+			/* This case is when we load a footprint with file->open, or from the command line */
 			CreateNewPCBPost (yyPCB, 0);
 			ParseGroupString("1,c:2,s", &yyPCB->LayerGroups, &yyData->LayerN);
 			e = yyPCB->Data->Element->data; /* we know there's only one */
@@ -236,7 +239,7 @@ parsepcb
 			yyPCB->MaxWidth = e->BoundingBox.X2;
 			yyPCB->MaxHeight = e->BoundingBox.Y2;
 			yyPCB->is_footprint = 1;
-		      }
+		      } else printf ("HELLO\n");
 		  }
 		;
